mokapot/mokapot.py

```diff
@@ -82,9 +96,30 @@ def main():
                 dataset.add_proteins(proteins)
 
     # Define a model:
+    model = None
     if config.load_models:
         model = [load_model(model_file) for model_file in config.load_models]
-    else:
+    elif enabled_plugins:
+        plugin_models = {}
+        for plugin_name, plugin in enabled_plugins.items():
+            model = plugin.get_model(config)
+            if model is not None:
+                logging.debug(f"Loaded model for {plugin_name}")
+                plugin_models[plugin_name] = model
+
+        if len(plugin_models) == 0:
+            msg = "No models were defined by plugins. Using default model."
+            logging.debug(msg)
+            model = None
+        else:
+            first_mod_name = list(plugin_models.keys())[0]
+            if len(plugin_models) > 1:
+                msg = "More than one model was defined by plugins."
+                msg += f" Using the first one. ({first_mod_name})"
+                logging.warning(msg)
```

Comment:
**nitpik:** I'd instead put the full string inside the warning:
```python
logging.warning(
    "More than one model was defined by plugins. Using the first one: %s",
    list(plugin_models.keys())[0]
)
```

Reply:
I did it that way because it exceeded the number of columns in the file and the alternatives were even uglier ... LMK what you preffer (I believe we should not exceed the 79 col line limit in this project)

Reply:
ok, found a way